extensions/brew/package.json

```
@@ -56,6 +56,14 @@
       "required": false,
       "title": "Custom Brew Executable Path",
       "description": "Set this if your brew executable is in a non-standard location"
+    },
+    {
+      "name": "zapCask",
+      "type": "checkbox",
+      "required": false,
+      "title": "Zap Casks",
+      "label": "Add --zap to brew uninstall --cask",
```

I'd suggest using a similar summary as `man brew` here and using a label like "Uninstall all files associated with a cask". This is clearer for non-technical users. The description text already explains the preference in more detail.

Probably worth also adding the warning "May remove files which are shared between applications" to the description?

---

> I'd suggest using a similar summary as `man brew` here and using a label like "Uninstall all files associated with a cask". This is clearer for non-technical users. The description text already explains the preference in more detail.

```suggestion
      "label": "Remove all files associated with a cask.",
```
Hi good suggestion! I'll make the change you can see here.

---

I've also updated the description with what I hope is clear enough.